### PR TITLE
CASSANDRA-18744 4.0: cassandra-stress in simplenative mode and prepared fails with exceptions

### PR DESCRIPTION
**Pull request by dmi0**
Thanks for sending a pull request! Here are some tips if you're new here:
 
 * Ensure you have added or run the [appropriate tests](https://cassandra.apache.org/_/development/testing.html) for your PR.
 * Be sure to keep the PR description updated to reflect all changes.
 * Write your PR title to summarize what this PR proposes.
 * If possible, provide a concise example to reproduce the issue for a faster review.
 * Read our [contributor guidelines](https://cassandra.apache.org/_/development/index.html)
 * If you're making a documentation change, see our [guide to documentation contribution](https://cassandra.apache.org/_/development/documentation.html)
 
Commit messages should follow the following format:

```
<One sentence description, usually Jira title or CHANGES.txt summary>

<Optional lengthier description (context on patch)>

patch by <Authors>; reviewed by <Reviewers> for CASSANDRA-#####

Co-authored-by: Name1 <email1>
Co-authored-by: Name2 <email2>

```

The [Cassandra Jira](https://issues.apache.org/jira/projects/CASSANDRA/issues/)

### CODE REVIEW

File: tools/bin/cassandra.in.sh [hunk 1]
```diff
@@ -31,7 +31,7 @@ CLASSPATH="$CASSANDRA_CONF"
 # it's just used here in constructing the classpath.
 if [ -d $CASSANDRA_HOME/build ] ; then
     #cassandra_bin="$CASSANDRA_HOME/build/classes/main"
-    cassandra_bin=`ls -1 $CASSANDRA_HOME/build/apache-cassandra*.jar`
+    cassandra_bin=`ls -1 $CASSANDRA_HOME/build/apache-cassandra*.jar | paste -sd ":" -`
```
Comment (smiklosovic) [inline tools/bin/cassandra.in.sh]: what is this `paste`? @dmi0

Comment (dmi0) [reply tools/bin/cassandra.in.sh]: `ls -1 $CASSANDRA_HOME/build/apache-cassandra*.jar` produces a list of files separated by "\n", which results in the `cassandra_bin` variable omitting some jar files. This issue caused the unit test to fail.

[ `paste -sd ":"`](https://linuxcommandlibrary.com/man/paste) joins all the file names into a single line, using ":" as a delimiter.

@smiklosovic

Comment (smiklosovic) [reply tools/bin/cassandra.in.sh]: wow ... could you please put here example of `cassandra_bin` with and without this change?

Comment (smiklosovic) [reply tools/bin/cassandra.in.sh]: All I see is this:

    ls -1 build/apache-cassandra*.jar | paste -sd ":" -
    build/apache-cassandra-5.0-beta1-SNAPSHOT.jar:build/apache-cassandra-5.0-beta1-SNAPSHOT-sources.jar

I do not think we need source on the class path? What additional jars do you see?

Comment (dmi0) [reply tools/bin/cassandra.in.sh]: Sorry, my bad. Just double checked, original one works for me as well (was confused by different jar versions)
Reverting this.

**Pull request closed without merging**